docs/UserGuide.md

```diff
@@ -234,6 +239,10 @@ Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEV
 <div markdown="block" class="alert alert-info">
 :information_source: <code>STUDENT_PHONE_NUMBER</code>, <code>GUARDIAN_PHONE_NUMBER</code> should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 and with 8 digits)
 </div>
+<div markdown="block" class="alert alert-primary">
+:bulb: TutorBuddy does not restrict the <code>STUDY_LEVEL</code> due to the vast number of education levels. The user 
```

Comment:
Hi, I was thinking if we could add in some example, "...number of education levels (E.g. Sec 2 / Secondary 2 / Computing / CS1101S)"

Reply:
Sure!